partner_aging/partner_aging_supplier.xml

```diff
@@ -0,0 +1,71 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+  <data>
+
+    <!-- Supplier Aging search/filters -->
+    <record model="ir.ui.view" id="view_supplier_aging_search">
+      <field name="name">supplier.aging.search</field>
+      <field name="model">partner.aging.supplier</field>
+      <field name="type">search</field>
+      <field name="arch" type="xml">
+        <search string="Supplier Aging">
+          <filter string="Overdue Only" name="overdue" help="Overdue" domain="[('max_days_overdue','&gt;',0),('total','&gt;',0)]"/>
+          <filter string="Credits Only" name="credits" help="Show unapplied supplier credits" domain="[('total','&lt;',0)]" />
+          <filter string="Exclude Credits" name="no_credits" help="Remove unapplied credits" domain="[('total','&gt;',0)]" />
+          <field name="partner_id" string="Supplier" />
+          <field name="invoice_id"/>
+          <field name="invoice_ref"/>
+          <field name="total" sum="Total"/>
+          <group expand="0" string="Group By...">
+            <filter string="Partner" domain="[]" context="{'group_by':'partner_id'}"/>
+            <filter string="Currency" domain="[]" context="{'group_by':'currency_name'}"/>
+          </group>
+        </search>
+      </field>
+    </record>
+
+    <!-- Supplier Aging Tree -->
+    <record model="ir.ui.view" id="view_supplier_aging_tree">
+      <field name="name">supplier.aging.tree</field>
+      <field name="model">partner.aging.supplier</field>
+      <field name="type">tree</field>
+      <field name="arch" type="xml">
+        <tree create="false" delete ="false" string="Aged Payables - Totals"
+              colors="red:max_days_overdue &gt; 0 and total &gt; 0;">
+          <field name="total" invisible="1"/>
+          <field name="partner_id" string="Supplier"/>
+          <field name="max_days_overdue"/>
+          <field name="date_due"/>
+          <field name="invoice_id"/>
+          <button name="open_invoice" type="object" icon="STOCK_INDEX" attrs="{'invisible':[('invoice_id','=',False)]}"/>
+          <field name="invoice_ref"/>
+          <field name="total" sum="Total"/>
+          <field name="current" string="Current" sum="Total"/>
+          <field name="days_due_01to30" sum="Total"/>
+          <field name="days_due_31to60" sum="Total"/>
+          <field name="days_due_61to90" sum="Total"/>
+          <field name="days_due_91to120" sum="Total"/>
+      <field name="days_due_121togr" sum="Total"/>
+      <field name="currency_name"/>
+          <field name="comment"/>
+        </tree>
+      </field>
+    </record>
+
+    <!-- Supplier Aging action -->
+    <record id="action_supplier_aging"
+            model="ir.actions.act_window">
+      <field name="name">Supplier Aging</field>
+      <field name="res_model">partner.aging.supplier</field>
+      <field name="view_type">form</field>
```

Comment:
Sorry, I got confused with the other deprecated type tags. Indeed this is needed. The previous 2 should be removed